### PR TITLE
Restore oedeploy tools to latest

### PR DESCRIPTION
**Pull request by CyanDevs**
- oeadmin uid/gid is now 1002:1002
- ansible updated to 2.9.18 from 2.8.8

### CODE REVIEW

**Review by rs-- — APPROVED** [review]

**Comment by ghost** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/openenclave/openenclave-ci/pull/28#" title="Author self-approved">CyanDevs</a>*, *<a href="https://github.com/openenclave/openenclave-ci/pull/28#pullrequestreview-604507515" title="Approved">rs--</a>*

The full list of commands accepted by this bot can be found [here](?repo=openenclave%2Fopenenclave-ci).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/openenclave/openenclave-ci/blob/master/OWNERS)~~ [rs--]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
